time logTime
uint8 level
string message
zoo_bot/GeoPosition robotPos